Android.bp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Android Open Source Project
+
+cc_defaults {
+    name: "exfat-utils-defaults",
+
+    cflags: [
+        "-DEXFAT_TOOLS_VERSION=\"1.0.1\"",
```

Comment:
I think EXFAT_TOOLS_VERSION should not be defined here. EXFAT_TOOLS_VERSION is generated automatically in config.h. So it is better to include config.h.

Reply:
And Could you add "-Wall" and "-Werror"?

Reply:
We can't generate config.h at runtime.
As far as I can see it isn't included in any .c/.h but only via c flags, this isn't the standard way.
As long we have sane defaults the required flags shall go there
-Wall and -Werror and implied by the android build system, if you want I can explicitly add them.